lib/cacheable.rb

```diff
@@ -1,22 +1,24 @@
 require 'cacheable/middleware'
-require 'cacheable/railtie'
+require 'cacheable/railtie' if defined?(Rails)
 require 'cacheable/response_cache_handler'
-require 'cacheable/controller'
 require 'msgpack'
 
 module Cacheable
 
-  def self.cache_store=(store)
-    @cache_store = nil
-    @store=store
+  def self.cache_store=(cache_store)
+    @cache_store = cache_store
   end
 
   def self.cache_store
-    @cache_store ||= ActiveSupport::Cache.lookup_store(*@store || Rails.cache)
```

Comment:
I think this is a breaking change for consumers of the gem/fork that rely on this behaviour. It's probably a limited number and maybe none does, but I'm thinking we should probably bump the gem's version

Reply:
Yes, this will probably be worth a major bump.